tasks/plugins.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Find plugins to install
+  find:
+    paths: "{{ local_plugins_dir }}"
+    file_type: directory
+    patterns: "({{fusiondirectory_plugins | join('|') }})"
+    use_regex: True
+  become: no
+  run_once: true
+  delegate_to: localhost
+  register: plugin_paths
+
+- name: Create plugin dir
+  file:
+    path: "{{ local_download_dir }}/fd-plugins"
+    state: directory
+  become: no
+  run_once: true
+  delegate_to: localhost
+
+- name: Copy plugins
+  command: rsync -varuEAXH --delete "{{ item }}/" "{{ local_download_dir }}/fd-plugins/{{ item | basename }}/"
+  become: no
+  run_once: true
+  delegate_to: localhost
+  register: out
+  changed_when: false
+  with_items: "{{ plugin_paths.files | map(attribute='path') | list }}"
+
+- name: Compress plugins
+  archive:
+    path: "{{ local_download_dir }}/fd-plugins"
+    dest: "{{ local_download_dir }}/fd-plugins.tar.gz"
+  become: no
+  run_once: true
+  delegate_to: localhost
+
+- name: Create temporary plugin directory
+  tempfile:
+    state: directory
+    suffix: .fd-plugs
+  changed_when: false
+  register: plugin_tmpfile
+
+- name: Copy plugins
+  copy:
+    src: "{{ local_download_dir }}/fd-plugins.tar.gz"
+    dest: "{{ plugin_tmpfile.path }}/"
+  changed_when: false
+
+- name: Install plugins
+  shell: "echo '{{ plugin_tmpfile.path }}/fd-plugins.tar.gz' | fusiondirectory-setup --set-fd_home=/usr/share/webapps/fusiondirectory --install-plugins --update-locales --update-cache"
+  changed_when: false
```

Comment:
can this setup command be executed multiple times without changes or sideeffects?